### DISCUSSION TITLE
<cbc:ActivityType>railway services</cbc:ActivityType>

### DISCUSSION THREAD

**Discussion by troa1** [OP-TED/eForms-SDK#84] [Q&A] [answered]
Message from our coders: 

examples/notices/veat_25.xml

 <cac:ContractingActivity>
    <cbc:ActivityType>railway services</cbc:ActivityType>
</cac:ContractingActivity>

 ActivityType doesn't exist in fields.json. Should this be ActivityTypeCode?
 
_______

I think this is related to the same issue I raised earlien in the discussion: BTs are missing links to codelists. At least that's what I think...

**Comment by bertrand-lorentz** [comment, marked as answer]
Thank you for your feedback.

This is a mistake in the example notice veat_25.xml. It should indeed use `cbc:ActivityTypeCode`, as follows:

```
<cac:ContractingActivity>
	<cbc:ActivityTypeCode listName="entity-activity">rail</cbc:ActivityTypeCode>
</cac:ContractingActivity>
```

This also requires the correction of the "Buyer Legal Type" (BT-11), otherwise the value "rail" is not allowed. For example:

```
<cac:ContractingPartyType>
	<cbc:PartyTypeCode listName="buyer-legal-type">pub-undert-cga</cbc:PartyTypeCode>
</cac:ContractingPartyType>
```

The example will be corrected in the next version of the SDK.
